pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -272,15 +281,15 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication {
     }
 
     /**
-     * @dev Sets a new root address
+     * @dev Sets the root address to `newRoot`.
      */
     function setRoot(address newRoot) external onlyExecutor {
         root = newRoot;
         emit RootSet(newRoot);
     }
 
     /**
-     * @dev Schedules a root change call
+     * @dev Schedules an execution to change the root address to `newRoot`.
      */
     function scheduleRootChange(address newRoot, address[] memory executors)
```

Comment:
It's not clear to me why `SCHEDULE_DELAY_ACTION_ID` is included here - I thought it was only applicable to `scheduleDelayChange`.

Reply:
The first argument of `scheduleWithDelay` is only emitted in an event - it is otherwise not used. It simply infroms that a delay change has been scheduled for `this.setRoot`. But it does look a bit wonky.